connect/src/main/java/com/stripe/android/connect/EmbeddedComponentManager.kt

```diff
@@ -1,21 +1,19 @@
 package com.stripe.android.connect
 
 import android.content.Context
-import android.os.Parcelable
 import androidx.activity.ComponentActivity
 import androidx.annotation.RestrictTo
 import androidx.fragment.app.FragmentActivity
 import com.stripe.android.connect.appearance.Appearance
 import com.stripe.android.connect.appearance.fonts.CustomFontSource
 import com.stripe.android.connect.di.StripeConnectComponent
 import com.stripe.android.connect.manager.EmbeddedComponentCoordinator
-import kotlinx.parcelize.Parcelize
 
 @PrivateBetaConnectSDK
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class EmbeddedComponentManager @JvmOverloads constructor(
-    configuration: Configuration,
-    fetchClientSecretCallback: FetchClientSecretCallback,
+    publishableKey: String,
```

Comment:
This might be difficult for merchants to do correctly.

We expect merchants to initialize this onCreate, but they might need to make a network call to get their publishable key.

While it's safe to deploy an app with the publishable key hard coded, some merchants support multiple merchants/countries/currencies.

Reply:
> We expect merchants to initialize this onCreate

That's not true. There is an `EmbeddedComponentManager.onActivityCreate()` method, but that's static and doesn't require any configuration. The `EmbeddedComponentManager` instance can be created whenever. In fact, we fetch the pk in the example app.